metadata-models/src/main/pegasus/com/linkedin/identity/CorpUserInfo.pdl

```diff
@@ -90,6 +91,7 @@ record CorpUserInfo includes CustomProperties {
   @Searchable = {
     "fieldType": "TEXT_PARTIAL",
     "queryByDefault": true,
+    "enableAutocomplete": true,
```

Comment:
Was this removed? I recall we used to have this

Reply:
I didn't remove it, so makes sense to add it, especially given that I am suggesting enabling autocomplete for users/groups. Also I added tests for autocomplete on users/groups to ensure it works as expected.